components/identity-core/org.wso2.carbon.identity.core/src/test/java/org/wso2/carbon/identity/core/SAMLSSOServiceProviderManagerTest.java

```diff
@@ -0,0 +1,356 @@
+        }
+    }
+
+    private SAMLSSOServiceProviderDO createServiceProviderDO(String issuer) {
+
+        SAMLSSOServiceProviderDO serviceProviderDO = new SAMLSSOServiceProviderDO();
+        serviceProviderDO.setIssuer(issuer);
+        serviceProviderDO.setIssuerQualifier(ISSUER_QUALIFIER);
+        serviceProviderDO.setAssertionConsumerUrls(ASSERTION_CONSUMER_URLS);
+        serviceProviderDO.setDefaultAssertionConsumerUrl(DEFAULT_ASSERTION_CONSUMER_URL);
+        serviceProviderDO.setCertAlias(CERT_ALIAS);
+        serviceProviderDO.setSloResponseURL(SLO_RESPONSE_URL);
+        serviceProviderDO.setSloRequestURL(SLO_REQUEST_URL);
+        serviceProviderDO.setDoSingleLogout(DO_SINGLE_LOGOUT);
+        serviceProviderDO.setDoSignResponse(DO_SIGN_RESPONSE);
+        serviceProviderDO.setDoSignAssertions(DO_SIGN_ASSERTIONS);
+        serviceProviderDO.setAttributeConsumingServiceIndex(ATTRIBUTE_CONSUMING_SERVICE_INDEX);
+        serviceProviderDO.setRequestedAudiences(REQUESTED_AUDIENCES);
+        serviceProviderDO.setRequestedRecipients(REQUESTED_RECIPIENTS);
+        serviceProviderDO.setEnableAttributesByDefault(ENABLE_ATTRIBUTES_BY_DEFAULT);
+        serviceProviderDO.setNameIDFormat(NAME_ID_FORMAT);
+        serviceProviderDO.setIdPInitSSOEnabled(IS_IDP_INIT_SSO_ENABLED);
+        serviceProviderDO.setIdPInitSLOEnabled(IDP_INIT_SLO_ENABLED);
+        serviceProviderDO.setIdpInitSLOReturnToURLs(IDP_INIT_SLO_RETURN_TO_URLS);
+        serviceProviderDO.setDoEnableEncryptedAssertion(DO_ENABLE_ENCRYPTED_ASSERTION);
+        serviceProviderDO.setDoValidateSignatureInRequests(DO_VALIDATE_SIGNATURE_IN_REQUESTS);
+        serviceProviderDO.setDoValidateSignatureInArtifactResolve(DO_VALIDATE_SIGNATURE_IN_ARTIFACT_RESOLVE);
+        serviceProviderDO.setSigningAlgorithmUri(SIGNING_ALGORITHM_URI);
+        serviceProviderDO.setDigestAlgorithmUri(DIGEST_ALGORITHM_URI);
+        serviceProviderDO.setAssertionEncryptionAlgorithmUri(ASSERTION_ENCRYPTION_ALGORITHM_URI);
+        serviceProviderDO.setKeyEncryptionAlgorithmUri(KEY_ENCRYPTION_ALGORITHM_URI);
+        serviceProviderDO.setAssertionQueryRequestProfileEnabled(IS_ASSERTION_QUERY_REQUEST_PROFILE_ENABLED);
+        serviceProviderDO.setSupportedAssertionQueryRequestTypes(SUPPORTED_ASSERTION_QUERY_REQUEST_TYPES);
+        serviceProviderDO.setEnableSAML2ArtifactBinding(ENABLE_SAML2_ARTIFACT_BINDING);
+        serviceProviderDO.setSamlECP(SAML_ECP);
+        serviceProviderDO.setIdpEntityIDAlias(IDP_ENTITY_ID_ALIAS);
+        serviceProviderDO.setDoFrontChannelLogout(DO_FRONT_CHANNEL_LOGOUT);
+        serviceProviderDO.setFrontChannelLogoutBinding(FRONT_CHANNEL_LOGOUT_BINDING);
+
+        return serviceProviderDO;
+    }
+
+    public String getIssuerWithQualifier(String issuer) {
+
+        return StringUtils.isNotBlank(ISSUER_QUALIFIER) ?
+                issuer + IdentityRegistryResources.QUALIFIER_ID + ISSUER_QUALIFIER : issuer;
+    }
+}
```

Comment:
Add a new line

Reply:
Resolved in [Introduce JDBC based persistence for SAML](https://github.com/wso2/carbon-identity-framework/pull/6241/commits/d1e67ce649f7a5d2e7e62f364ce93815f347e7b4)